### DISCUSSION TITLE
Support for Login with Unstoppable

### DISCUSSION THREAD

**Discussion by wendy-unstoppable** [rainbow-me/rainbowkit#755] [Ideas]
Login with Unstoppable allows an Unstoppable Domain owner to connect their wallet and permission the sharing information with applications, which works for Polygon and Ethereum domains.
This allows dapps to learn information about their users (email address or social profiles, such as Twitter handle) without needing to host or maintain their own CRM database.
With current wallet-based sign-in methods, it is difficult or impossible for dapps to contact and engage with their users.
It'd be awesome if Rainbowkit supported Login with Unstoppable!
https://docs.unstoppabledomains.com/login-with-unstoppable/

**Comment by qrtp** [comment]
Hello there, the UD dev team would be happy to raise a PR to add Login with Unstoppable support. It would be a nice compliment the existing SIWE package and enable apps to reach more web3 users. We just want to confirm the community is supportive of a PR before we raise one. Thank you!
